target/ppc/insn64.decode

```diff
@@ -94,23 +94,23 @@ XXSPLTI32DX     000001 01 0000 -- -- ................ \
 ## Vector GER instruction
 
 PMXVI4GER8      000001 11 1001 -- - - pmsk:8 xmsk:4 ymsk:4              \
```

Comment:
Sorry for not spotting this in the first pass, but maybe we should move the field definitions of xmsk and ymsk, which never change in MMIRR:XX3, to @MMIRR_XX3. And, in the instructions, keep only pmsk, which varies in size.